src/app/Http/Requests/StorePlaceRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StorePlaceRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'lat' => 'required|numeric',
```

Comment:
Use coordinate validation rule (the one you created last year 😉 )
(check event validation rules)

\App\Http\Livewire\Forms\Coordinates::rules("coordinates")